setup.py

```diff
@@ -1,11 +1,14 @@
 from setuptools import setup
 
+# Version comes directly from source w/o importing
+exec(open('closeio_api/_version.py').read())
```

Comment:
Shouldn't use `exec`, and the current directory may not necessarily be the one that contains `closeio_api`. Is there any convention on how to do this best?